README.md

```diff
@@ -3,16 +3,16 @@
 
```

Comment:
After running `TestXmlEncodingBased` can see following special characters. 
1. Please ensure source VAST XML and 
2. Generated VAST XML are syntactically valid documents
```sh
&#x9;&#x9;<Creative xmlns="http://www.iab.com/VAST" id="5480" sequence="1" adId="2447226">
&#x9;&#x9;  <UniversalAdId xmlns="http://www.iab.com/VAST" idRegistry="Ad-ID" idValue="8465">8465</UniversalAdId>
&#x9;&#x9;  <NonLinearAds xmlns="http://www.iab.com/VAST">
&#x9;&#x9;    <NonLinear xmlns="http://www.iab.com/VAST">
&#x9;&#x9;    <StaticResource xmlns="http://www.iab.com/VAST" creativeType="image/png">
&#x9;&#x9;    http://mms.businesswire.com/media/20150623005446/en/473787/21/iab_tech_lab.jpg
&#x9;&#x9;    </StaticResource>
&#x9;&#x9;    <NonLinearClickThrough xmlns="http://www.iab.com/VAST">
&#x9;&#x9;    http://iabtechlab.com
&#x9;&#x9;    </NonLinearClickThrough>
&#x9;&#x9;    <NonLinearClickTracking xmlns="http://www.iab.com/VAST">
&#x9;&#x9;    http://example.com/trackingurl/clickTracking
&#x9;&#x9;    </NonLinearClickTracking>
&#x9;&#x9;   </NonLinear>
&#x9;&#x9;   <TrackingEvents xmlns="http://www.iab.com/VAST">
&#x9;  &#x9;   <Tracking event="shriprasad"><![CDATA[https://mytracker.com]]></Tracking></TrackingEvents>
&#x9;  &#x9;  </NonLinearAds>
&#x9;&#x9;</Creative>
```

Reply:
Done